src/commands/txCommands/pullAll.js

```diff
@@ -6,39 +6,47 @@ const gitHelpers = require('./util/gitHelpers');
 
 const getProjectResourcesList$ =
 	txlib.resources$
+		// We want to sort the array of resources so that the ALL_TRANSLATIONS_RESOURCE is
+		// downloaded first, this will allow other resources to be applied on top of
+		// any changes in that resource. Hopefully, this should prevent any changes in
+		// feature branches from being overridden by this resource
+		.map(resource =>
+			resource
+				.sort((a, b) => a === txlib.ALL_TRANSLATIONS_RESOURCE ? -1 : 1)
```

Comment:
if we ever apply airbnb linting to this repo it will complain about the unused `b` arg